py/ngff_zarr/v06/zarr_metadata.py

```
@@ -0,0 +1,431 @@
+            
+            coordinate_systems.append(
+                CoordinateSystem(
+                    name=cs["name"],
+                    axes=axes
+                )
+            )
+
+        if not coordinate_systems:
+             raise ValueError(
+                 "Invalid OME-Zarr v0.6 metadata:" \
+                 " missing 'coordinateSystems' in multiscales metadata."
```

Ruff (0.15.17)

[warning] Implicitly concatenated string literals over multiple lines

(ISC002)